### PR TITLE
Alfred added to tools page

### PR DESCRIPTION
**Pull request by rohanpai**
*AUTHOR: Describe this change. Include screenshots if possible.*
![screen shot 2016-06-13 at 11 53 12 pm]

##### Reviewer tasks

*AUTHOR: Add other outstanding tasks for the reviewer. Remove irrelevant items below.*

- [ ] HACKS: reviewer approves hacks introduced by this change
- [ ] [OPS](./checklist.md#ops): reviewer verified correct monitoring/alerts were added
- [ ] METRICS: reviewer verified correct usage metric tracking was added

##### Test plan

- [ ] Run locally
- [ ] Push to staging box for testing
- [ ] Copy approved
- [ ] Verified that on staging box metrics worked
- [ ] Check on mobile devices and on small screens things don't mess up

### CODE REVIEW

File: app/web_modules/sourcegraph/home/ToolsHomeComponent.js [hunk 1]
```diff
@@ -130,6 +140,17 @@ class ToolsHomeComponent extends Component {
 		window.location.assign("https://github.com/sourcegraph/sourcegraph/tree/master/client/emacs");
 	}
 
+	_alfredPackalCTAClicked() {
+		this.context.eventLogger.logEventForPage("SourcegraphAlfredCTAClicked", "DashboardTools", {medium: "Packal"});
```
Comment (rohanpai) [inline app/web_modules/sourcegraph/home/ToolsHomeComponent.js]: I'm not sure if this is a correct way to log events on the webpage. 

Can anyone verify?

Comment (sqs) [inline app/web_modules/sourcegraph/home/ToolsHomeComponent.js]: What's Packal?

Comment (rohanpai) [inline app/web_modules/sourcegraph/home/ToolsHomeComponent.js]: Packal is the app store for Alfred Workflows (http://www.packal.org/)

**Comment by uforic** [comment]
Yo! First thing, for large PR's, make sure to remove the checklist items that don't apply. Remove the italicized text, and write out a test plan. In this case, the test plan might be:

- [ ] Run locally
- [ ] Push to staging box for testing
- [ ] Copy approved
- [ ] Verified that on staging box metrics worked
- [ ] Check on mobile devices and on small screens things don't mess up

or something.

File: app/web_modules/sourcegraph/home/ToolsHomeComponent.js [hunk 2]
```diff
@@ -130,6 +130,10 @@ class ToolsHomeComponent extends Component {
 		this.context.eventLogger.logEventForCategory(AnalyticsConstants.CATEGORY_TOOLS, AnalyticsConstants.ACTION_CLICK, "SourcegraphLiveCTAClicked", {page_name: AnalyticsConstants.PAGE_TOOLS, editorType: AnalyticsConstants.INTEGRATION_EDITOR_VIM});
 		window.location.assign("https://github.com/sourcegraph/sourcegraph-vim");
 	}
+	_alfredPackalCTAClicked() {
+		this.context.eventLogger.logEventForPage("SourcegraphAlfredCTAClicked", "DashboardTools", {medium: "Packal"});
+		window.location.assign("http://www.packal.org/workflow/sourcegraph");
```
Comment (uforic) [inline app/web_modules/sourcegraph/home/ToolsHomeComponent.js]: This action isn't needed anymore right?

File: app/web_modules/sourcegraph/home/ToolsHomeComponent.js [hunk 3]
```diff
@@ -144,6 +148,24 @@ class ToolsHomeComponent extends Component {
 		);
 	}
 
+	_alfredCTA() {
+		return (
+			<div>
+				<span className={base.ph1}>
```
Comment (uforic) [inline app/web_modules/sourcegraph/home/ToolsHomeComponent.js]: One button, not two needed there right?

File: app/web_modules/sourcegraph/util/EventLogger.js [hunk 4]
```diff
@@ -395,10 +395,15 @@ export function withViewEventsLogged(Component: ReactClass): ReactClass {
 				referred_by_sourcegraph_editor?: string;
 				language?: string;
 			};
-
+			
 			// TODO:matt remove this once all plugins are switched to new version
```
Comment (uforic) [inline app/web_modules/sourcegraph/util/EventLogger.js]: These comments were attached to what is now in the else-if, mind scooting them down a bit?

File: app/web_modules/sourcegraph/home/ToolsHomeComponent.js [hunk 5]
```diff
@@ -202,6 +224,20 @@ class ToolsHomeComponent extends Component {
 							</div>
 						</Panel>
 					</div>
+					<div styleName="panel-item">
+						<Panel hoverLevel="high">
+							<img styleName="img" src={`${this.context.siteConfig.assetsRoot}/img/Dashboard/AlfredLogo.png`}></img>
+							<Heading align="center" level="4" className={base.ph4}>For your launcher</Heading>
+							<p styleName="cool-mid-gray" className={base.ph4}>
+								Search Sourcegraph from Alfred.
```
Comment (uforic) [inline app/web_modules/sourcegraph/home/ToolsHomeComponent.js]: Maybe something a little more texty, at least so its two lines and fits in with the rest. Just one man's opinion. Maybe something like "Immediate access to Sourcegraph code search from Alfred,"

**Comment by uforic** [comment]
Ah, you'll notice checks have failed. Two ways to know about this before CircleCI tells you - check the browser logs, and you'll see there are warnings thrown by webpack about spacing in Javascript, and if you run this ```npm run check && npm run lint && npm run test-unit``` from within the sourcegraph/app folder, it would have failed.

One of those Sublime plugins does auto formatting of JS, or at least lets you know where the formatting is off.

**Comment by uforic** [comment]
LGTM! Except for more descriptive text. Before merging, make sure that the Alfred README is up to date, then go ahead and merge!

**Pull request merged**